recipe/meta.yaml

```diff
@@ -113,40 +104,41 @@ requirements:
     # GPU requirements
     - cudatoolkit {{ cudatoolkit }}*  # [pytorch_variant == "gpu"]
     - cudnn {{ cudnn }}*              # [pytorch_variant == "gpu"]
-    - magma                           # [pytorch_variant == "gpu"]
+    - magma 2.7.0                     # [pytorch_variant == "gpu"]
     # Required for GPU profiler
-    - cupti                           # [pytorch_variant == "gpu"]
+    - cupti 11.3.1                    # [pytorch_variant == "gpu"]
     # OpenBLAS or MKL
     - mkl-devel {{ mkl }}.*           # [blas_impl == "mkl"]
     - mkl {{ mkl }}.*                 # [blas_impl == "mkl"]
```

Comment:
The issue with `mkl` on `osx-64` isn't because we don't build it. It's because it was a pain in the butt trying to fix runtime issues caused by conflicts between `openblas` and `mkl`, and nobody was really asking for pytorch `mkl`+`osx-64`.

As an aside, `mkl` is available on `osx-64`, just not the most recent version. For my own understanding, would you mind explaining why this release would unblock?
```
dpetry@Daniels-MacBook-Pro ~/Sandbox/aggregate (master) $ conda search mkl[subdir=osx-64]
Loading channels: done
# Name                       Version           Build  Channel             
mkl                         2017.0.4      h1fae6ae_0  pkgs/main           
mkl                         2018.0.0      h5ef208c_6  pkgs/main           
mkl                         2018.0.0      hc285769_4  pkgs/main           
mkl                         2018.0.1      hfbd8650_4  pkgs/main           
mkl                         2018.0.2               1  pkgs/main           
mkl                         2018.0.3               1  pkgs/main           
mkl                           2019.0             117  pkgs/main           
mkl                           2019.0             118  pkgs/main           
mkl                           2019.1             144  pkgs/main           
mkl                           2019.3             199  pkgs/main           
mkl                           2019.4             233  pkgs/main           
mkl                           2020.0             166  pkgs/main           
mkl                           2020.1             216  pkgs/main           
mkl                           2020.2             260  pkgs/main           
mkl                         2021.2.0    hecd8cb5_269  pkgs/main           
mkl                         2021.3.0    hecd8cb5_517  pkgs/main           
mkl                         2021.4.0    hecd8cb5_637  pkgs/main           
mkl                         2022.0.0    hecd8cb5_105  pkgs/main           
mkl                         2022.1.0    hf3af037_208  pkgs/main           
mkl                         2022.1.0    hf3af037_209  pkgs/main
```

Reply:
Ah, thanks!